api-reference/endpoint/get-software.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: "Get Software Info"
+api: "POST https://eol.xeol.com/api/v1/query"
+description: "This endpoint gets a software's support information"
+---
+
+### Body
+
+<ParamField body="identifier" type="string[]" required="true">
+  List of software identifiers.
+  
+  Can be either in `cpe`, `purl`, or `custom` format.
+  
+  If `custom`, just use the software's name like this: `Red Hat Enterprise Linux`
+</ParamField>
+
+### Response
+
+<ResponseField name="results" type="object[]">
+  List of software with their listed supports
+
+  <Expandable title="result">
+    <ResponseField name="name" type="string">
+      Name of the matched artifact
```

Comment:
we should use consistent naming. either use entirely entity or entirely artifact. 

But artifact is not really descriptive since it could be a service like AWS RDS which we return information about.